llmfoundry/models/hf/hf_base.py

```diff
@@ -151,6 +151,7 @@ def build_config(
             trust_remote_code=trust_remote_code,
             use_auth_token=use_auth_token,
             attn_implementation=attn_implementation,
+            torch_dtype='float32',
```

Comment:
nit: could we make a global variable that is master weight precision?

Reply:
sure